Sem1Dz/main.py

```diff
@@ -0,0 +1,43 @@
+#Найдите сумму цифр трехзначного числа.
+print("Задача 1")
+n=int(input("Введите проверяемое число"))
+sum=0
+while n > 0:
+    digit = n % 10
+    sum += digit
+    n //= 10
+print(sum)
+
+
+
+#Задача про Катю, Сережу и Петю
+print("Задача 2")
+import math
+suma=int(input("Введите общее количество журавликов"))
+serg=petya=suma//6
+katya=2*(serg + petya)
```

Comment:
Во первых, вы забываете проверять входные данные на корректность. На семинаре я говорил, что это нужно делать, иначе мы можем получить нерелевантные результаты. Например, сумма полученных результатов может быть меньше, чем то число, которое было введено на входе. Во-вторых, почти нет пробелов.
```suggestion
suma = int(input("Введите общее количество журавликов"))
serg = petya = suma // 6
katya = 2 * (serg + petya)
```